Addons/Fbx.cs

```diff
@@ -71,12 +65,12 @@ static void TryLoadFbxSupport()
             var getMeshForComponent = FbxExporterAssembly.GetTypes()
                .Where(t => t.BaseType == typeof(MulticastDelegate) && t.Name.StartsWith("GetMeshForComponent"))
                .First(t => t.ContainsGenericParameters);
```

Comment:
We should replace the reflection here with an optional dependency on FBX Exporter